lib/wrappers/transform-node.js

```diff
@@ -13,6 +14,44 @@ module.exports = class TransformNodeWrapper extends NodeWrapper {
       cachePath: this.cachePath,
     });
     this.callbackObject = this.nodeInfo.getCallbackObject();
+    this.callbackObject.revised = () => {
```

Comment:
I don't think we want to expose `revised()` to the plugin initially, we should only be dealing with `volatile: true`